src/latbin/lattice-determinize-phone-pruned.cc

```diff
@@ -43,11 +44,13 @@ int main(int argc, char *argv[]) {
         "                            final.mdl ark:in.lats ark:det.lats\n";
 
     ParseOptions po(usage);
+    bool write_compact = true;
     BaseFloat acoustic_scale = 1.0;
     BaseFloat beam = 10.0;
     fst::DeterminizeLatticePhonePrunedOptions opts;
     opts.max_mem = 50000000;
 
+    po.Register("write-compact", &write_compact, "If true, write in normal (compact) form.");
```

Comment:
watch line length.  And add: "--write-compact=false allows you to retain frame-level score information."